api-reference/10 UI Components/dxDropDownButton/1 Configuration/deferRendering.md

```diff
@@ -6,6 +6,18 @@ default: true
 ---
 ##### shortDescription
 Specifies whether to wait until the drop-down menu is opened the first time to render its content.
+Specifies whether to render the view's content when it is displayed. If **false**, the content is rendered immediately.
 
 ---
-If this property is **false**, the content is rendered once the UI component is initialized.
\ No newline at end of file
+If this property is **false**, the content is rendered once the UI component is initialized.
+
+---
+##### Angular
+
+If you set the `deferRendering` property to **true**, wrap the required content in the Angular named [template](/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/#Templates).
```

Comment:
```suggestion
If you set the `deferRendering` property to **true**, wrap the content in the Angular [template](/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/#Templates).
```

I'm not sure why you add `required` since you cannot define 'required'. Why do you add `named` here?

Reply:
Apply the same changes to other parts